### PR TITLE
Add GetValue() for UserGroupExtensions

### PR DESCRIPTION
**Pull request by TheUltiOne**
Except this time it’s on the dev branch

### CODE REVIEW

File: Exiled.CustomItems/Exiled.CustomItems.csproj [hunk 1]
```diff
@@ -1,27 +1,28 @@
...
   <PropertyGroup>
     <OutputType>Library</OutputType>
     <AssemblyName>Exiled.CustomItems</AssemblyName>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <TargetFrameworkVersion>v2.0</TargetFrameworkVersion>
+    <FileUpgradeFlags>
+    </FileUpgradeFlags>
+    <UpgradeBackupLocation>
+    </UpgradeBackupLocation>
+    <OldToolsVersion>2.0</OldToolsVersion>
```
Comment (iopietro) [inline Exiled.CustomItems/Exiled.CustomItems.csproj:13]: All of this is not needed.

File: Exiled.CreditTags/Exiled.CreditTags.csproj [hunk 2]
```diff
@@ -1,26 +1,27 @@
...
     <Authors>Babyboucher20 &amp; iRebbok &amp; Exiled Team</Authors>
     <BaseProperties>false</BaseProperties>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <NoWarn>$(NoWarn);CS1591;SA1600</NoWarn>
+    <TargetFrameworkVersion>v2.0</TargetFrameworkVersion>
+    <FileUpgradeFlags>
+    </FileUpgradeFlags>
+    <UpgradeBackupLocation>
+    </UpgradeBackupLocation>
+    <OldToolsVersion>2.0</OldToolsVersion>
```
Comment (iopietro) [inline Exiled.CreditTags/Exiled.CreditTags.csproj:15]: Same as below.

File: EXILED.sln [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.30225.117
+# Visual Studio 15
+VisualStudioVersion = 15.0.28307.1300
```
Comment (iopietro) [inline EXILED.sln:4]: This is not needed.

File: Exiled.API/Extensions/UserGroupExtensions.cs [hunk 4]
```diff
@@ -38,5 +38,15 @@ public static bool EqualsTo(this UserGroup @this, UserGroup other)
         /// <returns>The key of that group, or null if not found.</returns>
         public static string GetKey(this UserGroup @this) => Server.PermissionsHandler._groups
             .FirstOrDefault(pair => pair.Value.EqualsTo(@this)).Key;
+
+        public static UserGroup GetValue(string GroupName)
+        {
+            var groups = ServerStatic.GetPermissionsHandler().GetAllGroups();
+            if (groups.ContainsKey(GroupName))
```
Comment (iopietro) [inline Exiled.API/Extensions/UserGroupExtensions.cs:45]: Can't you directly use `ServerStatic.GetPermissionsHandler().GetAllGroups().TryGetValue(...)`?

**Review by iopietro — CHANGES_REQUESTED** [review]

File: EXILED.sln (hunk 3 above)
Comment (TheUltiOne) [reply EXILED.sln:4]: Sorry, this was done automatically by my VisuAL sTUDIO. i GOTTA UPDATE TO 2019 LOL

Comment (iopietro) [reply EXILED.sln:4]: No problem, but you still have to revert it and the things I pointed out above.

Comment (TheUltiOne) [reply EXILED.sln:4]: Yeah sorry I'll do that right now

**Pull request closed without merging**